src/queries/Metadata.ts

```diff
@@ -44,3 +44,11 @@ export const MetadataRawQuery = gql`
         }
     }
 `;
+
```

Comment:
Hello, why do you introduce this query?
There is already query for this.
Use it.

Reply:
Where exactly can I find this query, because I couldn't find it myself.

Reply:
If you rebase to `master`, there's `MetadataQuery` in this very same file :))

Reply:
Oh, okay :smile: My bad.

Reply:
Done.